backend/packages/Upgrade/src/api/repositories/AnalyticsRepository.ts

```diff
@@ -418,100 +411,110 @@ export class AnalyticsRepository {
   }
 
   public async getCSVDataForSimpleExport(
+    experimentsData: any,
     experimentId: string,
+    isSRSExperiment: boolean,
     skip: number,
     take: number
   ): Promise<CSVExportDataRow[]> {
+    // Get the individual enrollment-related data
+    const individualEnrollmentRepository = getCustomRepository(IndividualEnrollmentRepository, 'export');
+    const individualEnrollmentQuery = individualEnrollmentRepository
+      .createQueryBuilder('individualEnrollment')
+      .select([
+        '"individualEnrollment"."userId" as "userId"',
+        '"individualEnrollment"."groupId" as "enrollmentGroupId"',
+        '"individualEnrollment"."enrollmentCode" as "enrollmentCode"',
+        '"individualEnrollment"."experimentId" as "experimentId"',
+        '"individualEnrollment"."conditionId" as "conditionId"',
+        '"individualEnrollment"."partitionId" as "partitionId"',
+      ]);
+
+    individualEnrollmentQuery.groupBy('individualEnrollment.userId')
+      .addGroupBy('individualEnrollment.groupId')
+      .addGroupBy('individualEnrollment.enrollmentCode')
+      .addGroupBy('individualEnrollment.experimentId')
+      .addGroupBy('individualEnrollment.conditionId')
+      .addGroupBy('individualEnrollment.partitionId')
+      .orderBy('individualEnrollment.userId', 'ASC')
+      .offset(skip)
+      .limit(take)
+      .where('individualEnrollment.experimentId = :experimentId', { experimentId });
+    
     const monitoredDecisionPointRepository = getCustomRepository(MonitoredDecisionPointRepository, 'export');
-    return monitoredDecisionPointRepository
+    const monitoredDecisionPointQuery = monitoredDecisionPointRepository
```

Comment:
This query needs to be run for users which you are getting from the above query otherwise, it will run the join for all the Monitored experiment tables which is not what we want.